gum/gum.c

```diff
@@ -506,6 +509,18 @@ gum_on_log_message (const gchar * log_domain,
 
     return;
   }
+#else
```

Comment:
It wouldn't make sense to have it output to console just in case people happen to be frida-ing a console-application?

I guess it makes sense to only have 1 definite path for those messages to come out.
Duplication is confusing.